app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    @authors = 5.times {@paper.authors.build}
```

Comment:
So `@authors = 5.times { @paper.authors.build }` stores `5` into the `@authors` instance variable